dev/fattest.simplicity/src/componenttest/topology/utils/tck/TCKResultsInfo.java

```diff
@@ -0,0 +1,124 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package componenttest.topology.utils.tck;
+
+/**
+ *
+ */
+public class TCKResultsInfo {
```

Comment:
Populate or remove Javadoc block (a one line description is probably all that's needed)